public/app/features/explore/TraceView/components/TracePageHeader/SpanFilters/SpanFilters.tsx

```diff
@@ -343,33 +336,41 @@ export const SpanFilters = memo((props: SpanFilterProps) => {
           </InlineField>
         </InlineFieldRow>
         <InlineFieldRow>
-          <InlineField label="Duration" labelWidth={16}>
-            <HorizontalGroup spacing={'xs'}>
+          <InlineField
+            label="Duration"
+            labelWidth={16}
+            tooltip="Filter by duration. Accepted units are ns, us, ms, s, m, h"
+          >
+            <HorizontalGroup spacing="xs" align="flex-start">
               <Select
                 aria-label="Select from operator"
                 onChange={(v) => setSpanFiltersSearch({ ...search, fromOperator: v.value! })}
                 options={[toOption('>'), toOption('>=')]}
                 value={search.fromOperator}
               />
-              <Input
-                aria-label="Select from value"
-                onChange={(v) => setSpanFiltersSearch({ ...search, from: v.currentTarget.value })}
+              <IntervalInput
+                ariaLabel="Select from value"
```

Comment:
I was a bit confused by this label. I was like select from which value? :D Would something like `Minimal span duration` work here?